### PR TITLE
RHDEVDOCS-4142: Adds Monitoring ConfigMap ref.

### PR DESCRIPTION
**Pull request by rkratky**
Version(s): 4.10

Issue: [RHDEVDOCS-4142](https://issues.redhat.com/browse/RHDEVDOCS-4142)

Link to docs preview: http://file.brq.redhat.com/rkratky/RHDEVDOCS-4142/monitoring/configmap-reference-for-cluster-monitoring-operator.html

Additional information: This is a temporary stop-gap fix for a customer-reported deficiency. The language has not been properly reviewed. Given this is reference material, we consider it OK for the time being. The entire reference content will be reviewed and updated ASAP.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@rkratky, just 4.10 and not 4.11, too?

File: monitoring/configmap-reference-for-cluster-monitoring-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+|Tolerations defines the Pods tolerations.
+
+|volumeClaimTemplate
+|link:https://github.com/prometheus-operator/prometheus-operator/blob/v0.57.0/Documentation/api.md#embeddedpersistentvolumeclaim[monv1.EmbeddedPersistentVolumeClaim]
+|VolumeClaimTemplate defines persistent storage for Alertmanager. It’s
+possible to configure storageClass and size of volume.
+|===
+
+
+== AlertmanagerUserWorkloadConfig
```
Comment (JoaoBraveCoding) [inline monitoring/configmap-reference-for-cluster-monitoring-operator.adoc:107]: We should remove this section since this is only supported in 4.11

**Review by JoaoBraveCoding — COMMENTED** [review]
Apart from the comment LGTM!

**Comment by rkratky** [comment]
> @rkratky, just 4.10 and not 4.11, too?

@kalexand-rh, yes. We'll have a better version for 4.12, so only merging this into 4.10 for now to respond to a customer request.

**Comment by kalexand-rh** [comment]
@sjstout and I agree that merging this change now and following up with edits later is the best way to meet the current customer need. @rkratky, let us know when you have the edits in.

**Comment by bergerhoffer** [comment]
@rkratky What is the status of these docs? It looks like they only went into 4.10 and are no longer in 4.11 or 4.12. Is that purposeful?

They're still hanging out in main though, so if they're no longer needed, we should remove that file in main (I'm hitting it in trying to bump kube instances of 1.24 to 1.25 in 4.12).

**Comment by bburt-rh** [comment]
@bergerhoffer - @rkratky can correct me if I'm wrong, but I believe that this is 4.10 only and can safely be removed from the main branch. I'm currently working on a new PR to add updated config map reference content in 4.11+.

**Comment by bergerhoffer** [comment]
That makes sense to me. @rkratky Any objections?

**Pull request merged**